packages/plugin-last-value/src/index.ts

```diff
@@ -1,6 +1,6 @@
 import deepmerge from 'deepmerge';
-import { addColumn } from '@/render';
-import { getLastReport, saveReport } from '@/storage';
+import { addColumn } from './render';
+import { getLastReport, saveReport } from './storage';
```

Comment:
Since this may be executed from a local/dev gimbal, the `@` path is used by gimbal and the `plugin-last-value`'s `tsconfig.json` will not be used so can only use relative paths. This is ok as the typescript paths for this plugin isn't really providing any benefit since it's a flat source directory.